src/main/main.cpp

```diff
@@ -75,6 +75,11 @@
 #include "daemon/DaemonManager.h"
 #endif
 
+#if defined(Q_OS_WIN)
+#include <QOpenGLContext>
+#include <QOpenGLFunctions>
```

Comment:
Is this include required?

Reply:
Yes. That's why i put it there.

Reply:
Ah, sorry. that was from another version of the patch. Can drop it now.

Reply:
Updated. Thanks